QualityUnit/liveagent-theme
# I2817 signup crm refactor

**Changes proposed in this Pull Request**
-- trial signup forms submission independent of crm javascript, submission and validation handled by backend,
-- after successfull response from crm api, account installation process is handled on /thank-you page,
-- thank you page with appropriate template is created automatically,
-- if crm script loaded, live validation of fields is processed by js too, for better ux,
-- refactored crm javascript, removed unused parts, removed dependency on jquery

-- fixed scripts lazy load for trial javascripts, script code has been strictly depended on jquery lib, after jquery removal didn't work properly and other non-jquery scripts were not loaded
-- fixed incorrect events listeners removal after scripts lazy load

**Testing instructions**
In order to create `thank-you` page automatically, necessary is visit of page as logged in user.  Script to check and create page doesn't run for not logged in users at all.

Close QualityUnit/web-issues#2817
Close QualityUnit/web-issues#2135
Close QualityUnit/web-issues#2873
